snabble/Android-SDK
# Add new payment status UI with addional feedback option

### What's new?

Changed payment status view UI and new feedback option. The User can now give additional feedback on the first two rating smileys. The happy smiley stays unchanged.

### UI Changes
* title moved to the top of the UI
* the rating layout is now part of a material card
* ratings images have been changed
* text input and send feedback button are adjustet to meet the requirements conditions
* the payment status view can be extended by implementing the 'snabble_external_feedback' layout

### How to test?

* deploy the application on the sdk branch 'new-feedback-option' 
* env staging/test -> check into a store -> buy something 
* on rating 1 or 2 click:
  * text input opens itself
  * it should be focused and the keyboard should open up
  * the send button shoudl be visible
  * on send click the rating ui shoudl vanish and the thanks message should appear
* on rating 3 click the behaviour should be the same as before
